spec/requests/submissions_spec.rb

```diff
@@ -53,6 +53,19 @@
         get submissions_phase_path(phase)
         expect(response).to have_http_status(:not_found)
       end
+
+      it "renders submission statistics" do
+        submission = create(:submission, challenge: challenge, phase: phase)
+        submission_2 = create(:submission, challenge: challenge, phase: phase, judging_status: "selected")
+
+
+        get submissions_phase_path(phase)
+        expect(response.body).to include("Boston Tea Party Cleanup")
+        # total submissions
+        expect(response.body).to include("2")
```

Comment:
this is not testing much since the expect below also has a 2 in it, but I found a way to use css selectors that gives more specific expectations. the trick is to `include Capybara::RSpecMatchers` in the request specs to enable the `have_css()` matcher. I'll add this now.